python/sglang/srt/model_executor/forward_batch_info.py

```diff
@@ -352,7 +352,10 @@ def init_new(
 
         if ret.forward_mode.is_idle():
             ret.positions = torch.empty((0,), device=device)
-            TboForwardBatchPreparer.prepare(ret)
+            if not model_runner.is_draft_worker:
```

Comment:
nit: shall we put this logic inside TboForwardBatchPreparer.prepare to avoid copy-paste the logic twice